src/Evolution/Systems/NewtonianEuler/Fluxes.hpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+// \cond
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace NewtonianEuler {
+
+/*!
+ * \brief Compute the fluxes of the conservative variables of the
+ * Newtonian Euler system
+ *
+ * The fluxes are \f$(\text{Dim} + 2)\f$ vectors of
+ * dimension \f$\text{Dim}\f$. Denoting the flux of the conservative
+ * variable \f$u\f$ as \f$F(u)\f$, one has
+ *
+ * \f{align*}
+ * F^i(\rho) &= S^i\\
+ * F^i(S^j) &= S^i v^j + \delta^{ij}p\\
+ * F^i(e) &= (e + p)v^i
+ * \f}
+ *
+ * wher \f$S^i\f$ is the momentum density, \f$e\f$ is the energy density,
+ * \f$v^i\f$ is the velocity, \f$p\f$ is the pressure, and \f$\delta^{ij}\f$
+ * is the Kronecker delta. This form of the fluxes combines conservative and
+ * primitive variables (while the velocity appears explicitly, the pressure
+ * implicitly depends, for instance, on the mass density and the specific
+ * internal energy), so the conversion from one variable set to the other
+ * must be known prior to calling this function.
+ */
+template <size_t Dim>
+void fluxes(gsl::not_null<tnsr::I<DataVector, Dim>*> mass_density_flux,
+            gsl::not_null<tnsr::IJ<DataVector, Dim>*> momentum_density_flux,
```

Comment:
Does momentum not have a lower index in the first position? Although it's Newtonian, so this might lead into a larger discussion that Will was thinking about on how to deal with Newtonian indices.